syntax = "proto3";

message TextInput {
  string id = 1;
  string label = 2;
  string value = 3;
  string default = 4;
}
